src/elements/content-sidebar/versions/VersionsItem.js

```diff
@@ -77,7 +77,7 @@ const VersionsItem = ({
     const isDeleted = action === VERSION_DELETE_ACTION;
     const isDownloadable = !!is_download_available;
     const isLimited = versionCount - versionInteger >= versionLimit;
-    const isRestricted = isWatermarked && !isCurrent && !can_download; // Watermarked files use can_download for preview
+    const isRestricted = isWatermarked && !isCurrent; // Watermarked files do not support prior version preview
```

Comment:
Didnt break test?

Reply:
The test is _also_ checking for the `can_download` permission, so it still passes. I'm hoping this change is temporary.